frontend/src/dmarc/DmarcReportPage.js

```diff
@@ -275,6 +275,7 @@ export default function DmarcReportPage() {
           frontendPagination={true}
           searchPlaceholder={t`Search DKIM Failing Items`}
           fileName={fileName}
+          onToggle={onToggle}
```

Comment:
I'm getting a delay when opening/closing the info panel on this page. I have a feeling it's due to the shared state causing the whole page to rerender. Might be worth letting each of the buttons handle their own state for the drawer (like GettingStarted). Not crucial though.